server/routes/courtRegister/allCourtsPagedView.ts

```diff
@@ -1,13 +1,28 @@
-import { CourtsPage } from '../../@types/courtRegister'
-import type { AllCourtsFilter, CourtsPageView } from './courtMapper'
-import { courtsPageMapper } from './courtMapper'
+import { CourtsPage, CourtType } from '../../@types/courtRegister'
+import { AllCourtsFilter, courtsPageMapper, CourtsPageView } from './courtMapper'
 
 export default class AllCourtsPagedView {
-  constructor(private readonly courtsPage: CourtsPage, private readonly filter: AllCourtsFilter) {}
+  constructor(
+    private readonly courtsPage: CourtsPage,
+    private readonly allCourtsFilter: AllCourtsFilter,
+    private readonly courtTypes: Array<CourtType>
+  ) {}
 
-  readonly courtsPageView: CourtsPageView = courtsPageMapper(this.courtsPage, this.filter)
+  readonly courtsPageViewDetails = courtsPageMapper(this.courtsPage, this.allCourtsFilter)
 
   get renderArgs(): CourtsPageView {
-    return this.courtsPageView
+    return {
+      ...this.courtsPageViewDetails,
+      allCourtsFilter: this.allCourtsFilter,
+      courtTypes: AllCourtsPagedView.sortCourtTypesByName(this.courtTypes),
+    }
+  }
+
+  private static sortCourtTypesByName(courtTypes: CourtType[]): CourtType[] {
```

Comment:
I used to use underscore for this kind of stuff.  Then lodash.  I wonder what the modern alternative is?